src/pegout/components/PegOutForm.vue

```diff
@@ -240,6 +240,7 @@ export default defineComponent({
         name: 'PegOutSuccess',
         params: {
           wallet: currentWallet.value,
+          type: 'powpeg',
```

Comment:
I don't know if there a file with common constants, but probably is good to include type as one of them, so we can use `constants.POWPEG` that will store `'powpeg'` string therefor we avoid typo errors.